airflow/cli/commands/config_command.py

```diff
@@ -15,15 +15,13 @@
 # specific language governing permissions and limitations
 # under the License.
 """Config sub-commands"""
+import io
+
 from airflow.configuration import conf
 
 
 def show_config(args):
```

Comment:
Could we remove unuse variable `args`?

Reply:
It is required by the function that calls these functions. Simply: All commands accept arguments. It's required.
https://github.com/apache/airflow/blob/master/airflow/bin/airflow#L39

Reply:
👌